benchmarks/nemo/tracer_advection/compute_in_subroutine/Makefile

```diff
@@ -55,7 +55,7 @@ tra_adv_acc: dl_timer
 # OpenACC version with nvtx profiling instrumentation.
 tra_adv_acc_prof: dl_timer
 	mkdir -p $@
-	${PSYCLONE} --profile invokes -s ../scripts/kernels_trans.py -opsy \
+	${PSYCLONE} --profile invokes -s ../scripts/acc_kernels_unified_memory_trans.py -opsy \
             $@/tra_adv_compute.f90 ./tra_adv_compute_auto_arrays.F90
 	cp Makefile_gen $@/Makefile
 	cp tra_adv_driver.F90 $@/.
```

Comment:
Is is probably because PSYCLONE_NVIDIA_LIB_DIR was not defined. I think this confirms that for this folder the nvidia_acc.sh was used.
I added a proper error reporting for this so at least it is clear the reason for failing.